src/platform_linux/shmem.c

```diff
@@ -718,6 +829,8 @@ platform_shm_track_free(shmem_heap *shm,
       debug_assert(frag->shm_frag_allocated_to_pid != 0);
       debug_assert(frag->shm_frag_size != 0);
 
+      shm->shm_num_frags_inuse--;
```

Comment:
Minor accounting bug in older code .... When a large fragment was freed, it was not being account for the 'in-use' counter.